test-cases/gutenberg/columns.md

```diff
@@ -275,3 +292,43 @@ Expected look:
 ![DarkModeEmpty](../resources/columns-dark-mode-empty.png)
 
 --------------------------------------------------------------------------------
+
+##### TC019
+
+### Columns layout picker flow
+
+- Add a `Columns` block
+- Expect to see bottom sheet with layout picker
+- Choose one of the options e.g: `33/33/33`
+- Select the first column
+- Open its settings
+- Change the value e.g. to  `90`
+- Expect that fill level in the preview (to the left of a slider) increased
+- Close bottom sheet and select the second column
+- Open its settings
+- Press text input (to the right of a slider)
+- Type value with more than 1 decimal e.g.: `55.55555` and press `Done`
+- Expect to have value with only 1 decimal
+- Close bottom sheet and switch to HTML mode
+- Expect that first column has width equal `90` and the second `55.5`
+- Save the post and open it on web
+- Expect to see columns displayed proportionally to the set width
```

Comment:
Too many "Expect"s in one test, could we maybe split this one? I see at least 3 different cases here:

- Check if preview icons are adjusting as we change column percentages
- Check if the column percentage value has proper rounding
- Check if column percentages are getting effective on frontend

Ideally I'd expect that if a test case fails we can link it to a bug report, so better to keep them more atomic.

Reply:
A correction for myself: I'd remove `Check if` statements, it is kinda redundant in this context.

Reply:
Done, please recheck that